lms/djangoapps/certificates/api.py

```diff
@@ -159,15 +159,16 @@ def generate_user_certificates(student, course_key, course=None, insecure=False,
 
     if not course:
         course = modulestore().get_course(course_key, depth=0)
-
+    # email will sent on completion of certificate generation to the student only if generate_pdf = True
```

Comment:
Can we avoid this core change?

Reply:
jysy khty hyn wysy kr lyty hyn

Reply:
We have minimized number of core changes, but few are still there.